processor/src/main/java/com/tickaroo/tikxml/processor/scanning/AnnotationScanner.kt

```diff
@@ -174,12 +174,12 @@ class AnnotationScanner(protected val elementUtils: Elements, protected val type
                 val getter = checkGetter(it, methodsMap, false)
                 val setter = checkSetter(it, methodsMap)
 
+                // Set access policy
```

Comment:
Why has this been moved before `if (it is NamedField)`

Reply:
If you run koltin tests without moving it the variable `accessResolver` will be not initliazed in call to addFieldToAnnotatedClass and processing annotations will halt with that error.

Reply:
Sweet! That's why I ❤️ integration tests. Thanks for fixing it!